scripts/superdesk/editor2/editor.js

```diff
@@ -1113,7 +1119,26 @@ angular.module('superdesk.editor2', [
                         updateTimeout = $timeout(vm.updateModel, 800, false);
                     });
 
+                    /**
+                     * Test if given point {x, y} is in given bouding rectangle.
+                     */
+                    function isPointInRect(point, rect) {
+                        return rect.left < point.x && rect.right > point.x && rect.top < point.y && rect.bottom > point.y;
+                    }
+
                     editorElem.on('contextmenu', function(event) {
+                        var err, pos;
+                        var point = {x: event.clientX, y: event.clientY};
+                        var errors = elem[0].parentNode.getElementsByClassName('sderror');
```

Comment:
This is also a live array, the length can't really be trusted/much slower to use than using slice to create a real array.